proto/decentraland/social/friendships/friendships.proto

```diff
@@ -85,30 +78,74 @@ message UpdateFriendshipPayload {
   optional Payload auth_token = 2;
 }
 
-message UpdateFriendshipResponse { FriendshipEventResponse event = 1; }
-
-message SubscribeFriendshipEventsUpdatesResponse {
-  repeated FriendshipEventResponse events = 1;
-}
-
 message Payload {
   // For internal use only, subject to change.
   optional string synapse_token = 1;
 }
 
+message BadRequestError {
+  string message = 1;
+}
+message UnauthorizedError {
+  string message = 1;
+}
+message ForbiddenError {
+  string message = 1;
+}
+message TooManyRequestsError {
+  string message = 1;
+}
+message InternalServerError {
+  string message = 1;
+}
+
+message UsersResponse {
+  oneof response {
+    Users users = 1;
+    InternalServerError internal_server_error = 2;
+    UnauthorizedError unauthorized_error = 3;
+  }
+}
```

Comment:
When trying to get_user_id_from_request, we go to [synapse](https://spec.matrix.org/v1.3/client-server-api/#get_matrixclientv3accountwhoami), which can return 200, 401, 403, or 429.

We map these responses [here](https://github.com/decentraland/social-service/blob/1cff62e924f3b5585285dcd5f546a649b4f55b5e/src/components/synapse.rs#L396-L415). Should we include them?

Reply:
So, we should include them all in all methods? As all of them are using the same auth mechanism 🤔